interfaces/prompting/requestprompts/export_test.go

```diff
@@ -33,12 +33,12 @@ const (
 	MaxOutstandingPromptsPerUser = maxOutstandingPromptsPerUser
 )
 
-func NewPrompt(id prompting.IDType, timestamp time.Time, snap string, iface string, path string, remainingPermissions []string, availablePermissions []string, originalPermissions []string) *Prompt {
+func NewPrompt(id prompting.IDType, timestamp time.Time, snap string, iface string, path string, outstandingPermissions []string, availablePermissions []string, originalPermissions []string) *Prompt {
```

Comment:
otherwise it barely fits the screen :smile: 
```suggestion
func NewPrompt(
	id prompting.IDType, timestamp time.Time, snap, iface, path string, 
	outstandingPermissions, availablePermissions, originalPermissions []string,
) *Prompt {
```